### PR TITLE
V2.0.0 rc2

### PR DESCRIPTION
**Pull request by caseydavenport**

### CODE REVIEW

File: v2.0/getting-started/kubernetes/installation/hosted/calico.yaml [hunk 1]
```diff
@@ -122,8 +122,6 @@ spec:
             # below.
             - name: NO_DEFAULT_POOLS
               value: "true"
-            - name: FELIX_LOGSEVERITYSCREEN
```
Comment (caseydavenport) [inline v2.0/getting-started/kubernetes/installation/hosted/calico.yaml]: This shouldn't go away! 

Must be missing in master too.

File: v2.0/getting-started/kubernetes/installation/hosted/k8s-backend/calico.yaml [hunk 2]
```diff
@@ -58,7 +59,8 @@ spec:
         # container programs network policy and routes on each
         # host.
         - name: calico-node
-          image: calico/node:v1.0.0-beta 
+          image: calico/node:v1.0.0-rc2 
+          imagePullPolicy: Always
```
Comment (caseydavenport) [inline v2.0/getting-started/kubernetes/installation/hosted/k8s-backend/calico.yaml]: This should be removed.

File: v2.0/getting-started/kubernetes/tutorials/simple-policy.md [hunk 3]
```diff
@@ -105,7 +105,7 @@ If you don't see a command prompt, try pressing enter.
 / # wget -q --timeout=5 nginx -O -
 ```
 
-However, we still cannot access the Service from a Pod without the label `run: access`:
+However, we still cannot access the Service from a Pod without the label `access`:
```
Comment (caseydavenport) [inline v2.0/getting-started/kubernetes/tutorials/simple-policy.md]: This is weird and wrong.

File: v2.0/getting-started/rkt/vagrant/index.md [hunk 4]
```diff
@@ -72,7 +72,7 @@ You can check that it's running
 
 	$ sudo rkt list
 	UUID            APP     IMAGE NAME                              STATE   NETWORKS
-	bc13af40        node    registry-1.docker.io/calico/node:v1.0.0-beta running
+	bc13af40        node    registry-1.docker.io/calico/node:v1.0.0-rc2 running
```
Comment (caseydavenport) [inline v2.0/getting-started/rkt/vagrant/index.md]: This really stinks.  The release tool messes up all of this output.

File: v2.0/getting-started/docker/tutorials/basic.md [hunk 5]
```diff
@@ -64,7 +64,7 @@ You should see output like this on each node
 
     vagrant@calico-01:~$ docker ps
     CONTAINER ID        IMAGE                COMMAND             CREATED             STATUS              PORTS               NAMES
-    408bd2b9ba53        calico/node:v1.0.0-beta   "start_runit"       About an hour ago   Up About an hour                        calico-node
+    408bd2b9ba53        calico/node:v1.0.0-rc2   "start_runit"       About an hour ago   Up About an hour                        calico-node
```
Comment (robbrockbank) [inline v2.0/getting-started/docker/tutorials/basic.md]: Need an issue to sort out table spacing :-)

File: v2.0/getting-started/kubernetes/installation/integration.md [hunk 6]
```diff
@@ -78,7 +78,23 @@ Requires=docker.service
 User=root
 Environment=ETCD_ENDPOINTS=http://<ETCD_IP>:<ETCD_PORT>
 PermissionsStartOnly=true
-ExecStart=/usr/bin/docker run --net=host --privileged --name=calico-node -e ETCD_ENDPOINTS= -e HOSTNAME=${HOSTNAME} -e IP= -e NO_DEFAULT_POOLS= -e AS= -e ETCD_AUTHORITY=127.0.0.1:2379 -e ETCD_SCHEME=http -e CALICO_LIBNETWORK_ENABLED=true -e IP6= -e CALICO_NETWORKING_BACKEND=bird -v /var/run/calico:/var/run/calico -v /lib/modules:/lib/modules -v /run/docker/plugins:/run/docker/plugins -v /var/run/docker.sock:/var/run/docker.sock -v /var/log/calico:/var/log/calico calico/node:v1.0.0-beta
+ExecStart=/usr/bin/docker run --net=host --privileged --name=calico-node
```
Comment (robbrockbank) [inline v2.0/getting-started/kubernetes/installation/integration.md]: Are you missing a line continuation on this line?

File: v2.0/getting-started/kubernetes/installation/integration.md [hunk 7]
```diff
@@ -78,7 +78,23 @@ Requires=docker.service
...
+-e ETCD_AUTHORITY=127.0.0.1:2379 \
+-e ETCD_SCHEME=http \
+-e CALICO_LIBNETWORK_ENABLED=true \
+-e IP6= \
+-e CALICO_NETWORKING_BACKEND=bird \
+-v /var/run/calico:/var/run/calico \
+-v /lib/modules:/lib/modules \
+-v /run/docker/plugins:/run/docker/plugins \
+-v /var/run/docker.sock:/var/run/docker.sock \
+-v /var/log/calico:/var/log/calico \
```
Comment (robbrockbank) [inline v2.0/getting-started/kubernetes/installation/integration.md]: Can we indent the continuing lines by a couple of spaces (makes it easier to read)

File: master/getting-started/rkt/vagrant/cloud-config/master-config.yaml [hunk 8]
```diff
@@ -28,8 +28,8 @@ coreos:
         ExecStartPre=/usr/bin/mkdir -p /opt/bin
         ExecStartPre=/usr/bin/mkdir -p /etc/rkt/net.d
         ExecStartPre=/usr/bin/wget -N -O /opt/bin/calicoctl http://www.projectcalico.org/builds/calicoctl
-        ExecStartPre=/usr/bin/wget -N -P /etc/rkt/net.d https://github.com/projectcalico/calico-cni/releases/download/v1.5.1/calico
-        ExecStartPre=/usr/bin/wget -N -P /etc/rkt/net.d https://github.com/projectcalico/calico-cni/releases/download/v1.5.1/calico-ipam
+        ExecStartPre=/usr/bin/wget -N -P /etc/rkt/net.d https://github.com/projectcalico/calico-cni/releases/download/v1.5.3/calico
```
Comment (caseydavenport) [inline master/getting-started/rkt/vagrant/cloud-config/master-config.yaml]: It should be easy enough to splat over these.  These changes all part of the automated release process - I think we should keep them and then just take your changes on top.

File: v2.0/getting-started/kubernetes/installation/integration.md (hunk 6 above)
Comment (caseydavenport) [inline v2.0/getting-started/kubernetes/installation/integration.md]: Wow yep, good spot.

File: v2.0/releases/index.md [hunk 9]
```diff
@@ -6,6 +6,21 @@ The following table shows component versioning for Calico  **{{ page.version }}*
...
+| calico/node                   | [v1.0.0-rc2](https://github.com/projectcalico/calico-containers/releases/tag/v1.0.0-rc2)   |
+| calico/cni                    | [v1.5.3](https://github.com/projectcalico/calico-cni/releases/tag/v1.5.3)                  |
+| libcalico                     | [v0.19.0](https://github.com/projectcalico/libcalico/releases/tag/v0.19.0)                 |
+| libcalico-go                  | [v1.0.0-rc4](https://github.com/projectcalico/libcalico-go/releases/tag/v1.0.0-rc4)        |
+| calico-bird                   | [v0.2.0-rc1](https://github.com/projectcalico/calico-bird/releases/tag/v0.2.0-rc1)         |
+| libnetwork-plugin             | [v1.0.0-rc3](https://github.com/projectcalico/libnetwork-plugin/releases/tag/v1.0.0-rc3)   |
+| calico/kube-policy-controller | [v0.5.1](https://github.com/projectcalico/k8s-policy/releases/tag/v0.5.1)                  |
+| networking-calico             | [889cfff](http://git.openstack.org/cgit/openstack/networking-calico/tree/?id=889cfff)      |
+
 ## v2.0.0-beta
```
Comment (robbrockbank) [inline v2.0/releases/index.md]: Do we want this here still?  I'm easy either way, but the 2.0 docs no longer refer to beta.

File: v2.0/getting-started/kubernetes/tutorials/simple-policy.md (hunk 3 above)
Comment (caseydavenport) [inline v2.0/getting-started/kubernetes/tutorials/simple-policy.md]: Yeah it is.

**Pull request merged**